DEVELOPMENT_GUIDE.MD

```diff
@@ -0,0 +1,102 @@
+# Getting Started with the Bootstrap-plus FitNesse Theme
+This MD is meant for people who want to develop for the Bootstrap-plus FitNesse Theme.
+
+## Running the code 
+
+Bootstrap+ is a maven project because it is a dependency of the Toolchain Plugin.
+Every maven project has a pom file that contains the version number of the pom and the dependencies.  
+
+Bootstrap+ can't be run on it's own. 
+To run Bootstrap+ it needs to be compiled within the Toolchain Plugin.
```

Comment:
Bootstrap+ can't be run on it's own. It is directly dependent on the Toolchain plugin. The Toolchain plugin in it's turn is a plugin to use in combination with FitNesse and HSAC. Run FitNesse with these dependencies in your dev environment. 

En even linkjes maken van Toolchain plugin, hsac en fitnesse.

Reply:
Tekst aangepast met de bijbehorende linkjes.